### ISSUE TITLE
Update Potassco Apps

### ISSUE THREAD

**Issue by susuhahnml** [potassco/potassco.github.io#60]
It is also hard to find, I am not sure if documentation is the best place.

**Comment by nrueh** [comment]
I agree, I think I have never seen this part before but it seems very relevant nowadays so maybe it should be on the front page even.
Also maybe it should be renamed to applications instead of apps?
